tests/test_config_utils.py

```diff
@@ -9,6 +10,12 @@
 ORIGINAL_CONFIG = copy.deepcopy(bigchaindb._config)
 
 
+@pytest.fixture
+def ignore_local_config_file(monkeypatch):
+    from bigchaindb.config_utils import file_config
+    monkeypatch.setattr('bigchaindb.config_utils.file_config', file_config)
```

Comment:
`pytest` documentation on overriding fixtures at the module level: http://doc.pytest.org/en/latest/fixture.html#override-a-fixture-on-a-test-module-level